next.config.mjs

```diff
@@ -1,5 +1,5 @@
+import nextPWA from '@ducanh2912/next-pwa';
```

Comment:
why use this package? Is it better than `next-pwa`?

Reply:
https://github.com/shadowwalker/next-pwa has not been actively maintained for months and does not work fine with next@14. Therefore, we need to switch to the forked version at https://github.com/DuCanhGH/next-pwa.

Reply:
Good catch